docs/src/content/docs/reference/configuration.mdx

```diff
@@ -208,7 +208,7 @@ interface BadgeConfig {
 
 ### `locales`
 
-**type:** <code>{ \[dir: string\]: [LocaleConfig](#localeconfig) }</code>
+**type:** <code>\{ \[dir: string\]: [LocaleConfig](#localeconfig) \}</code>
```

Comment:
Change required due to the MDX conversion to avoid interpreting this as an expression.